spec/acceptance/conversations_spec.rb

```diff
@@ -11,22 +11,26 @@
   let(:conversation) { create :conversation_with_messages, unread: true }
   let(:user) { conversation.initiator }
 
+  with_conversation_response = lambda {
```

Comment:
Never saw this kind of tests before 😱

Reply:
These tests are actually used to generate the api blueprint, nothing else.
And this lambda is to just call it inside the examples with this very same response.